.github/workflows/main.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           images: ${{ env.REGISTRY_IMAGE }}
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v3
+        if: ${{ !contains(matrix.platform, 'arm') && !contains(matrix.platform, 'amd64') }}
```

Comment:
@bendavies As you suggested, not installing QEMU during runs when the image we're building is for `arm64` or `amd64`. But still installing it for others, which will be never in this situation